pdu/ShortMessage.go

```diff
@@ -240,9 +240,18 @@ func (c *ShortMessage) Encoding() data.Encoding {
 	return c.enc
 }
 
+// String returns message content or content parse error
+func (c *ShortMessage) String() string {
```

Comment:
I dont think we need stringer interface for this case. Sm may be in wrong encoding. Override real value with error message is not what i want to commit for the library

Reply:
I think Stringer is not for purpose of debugging when it may contains error that must be handle. How about some method like: `Debug() string`

Reply:
```go
// the `pd` is `*pdu.DeliverSM` or `*pdu.SubmitSM`
fmt.Println(pd)
```

Reply:
`Debug() string` is not a good idea

https://github.com/linxGnu/gosmpp/blob/cc603b5cc822dcd18a9d1e72b65158f773be5db5/example/main.go#L78-L94

include this example program

Reply:
I still don't think it's a good way to log/print the debug. We can't sacrifice error checking for debugging purpose only. This case is hard, I think it's better to let user manually format its log!

Reply:
https://golang.org/pkg/fmt/#GoStringer

Reply:
There is no convention for using Stringer right? Especially when it may result in error. I still don't think we should do that. I want user understand what's going on.

Reply:
OK